source/Octopus.Tentacle/Services/Scripts/Kubernetes/KubernetesScriptServiceV1Alpha.cs

```diff
@@ -86,8 +86,7 @@ public async Task<KubernetesScriptStatusResponseV1Alpha> GetStatusAsync(Kubernet
             var trackedPod = podStatusProvider.TryGetTrackedScriptPod(request.ScriptTicket);
             return trackedPod != null
                 ? await GetResponse(trackedPod, request.LastLogSequence, cancellationToken)
-                //if we are getting the status of an unknown pod, return that it's still pending
-                : new KubernetesScriptStatusResponseV1Alpha(request.ScriptTicket, ProcessState.Pending, 0, new List<ProcessOutput>(), request.LastLogSequence);
+                : throw new Exception("Can't find Script Pod for: " + request.ScriptTicket);
```

Comment:
Yeah, I think this is ok 👍 

Can we just improve the error message, and maybe make a custom exception type as this will be reported back to the task log in Server, so might be nice to have a slightly friendlier message?

Reply:
![CleanShot 2024-04-15 at 16 30 55@2x](https://github.com/OctopusDeploy/OctopusTentacle/assets/2888279/d8faaefa-2b9d-404a-9b5a-657b300aec0b)

Reply:
like it!